website/docs/r/macie2_findings_filter.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Macie"
+layout: "aws"
+page_title: "AWS: aws_macie2_findings_filter"
+description: |-
+  Provides a resource to manage an AWS Macie Findings Filter.
```

Comment:
References to "AWS Macie" should be changed to "Amazon Macie" to match AWS' naming conventions